### ISSUE TITLE
Share the current rant to twitter instead of the website link

### ISSUE THREAD

**Issue by prodicus** [prodicus/trumporate#15]
Right now, the iteration allows you to share the website link on Twitter.

**Comment by riturajDas** [comment]
On it 👍🏽

**Comment by prodicus** [comment]
All right, assigning this to you then.

**Comment by riturajDas** [comment]
@oxalorg do u have any idea about manipulating the "href" attribute of the twitter-share button?
- I have tried loading in the current rant into the href attr using `$('.tweet').setAttribute('href','https://*'+rant+'*');`
- SO suggests such ways are possible but isn't working on mine. Need some regex for " "(space) and "."(dot/fullstop)?

**Comment by oxalorg** [comment]
@riturajDas 

Are you looking for something like this:

```
$('.tweet').attr('href', 'https://domain.com/'+ encodeURIComponent(rant));
```

**Comment by riturajDas** [comment]
@oxalorg hmm correct, so how do we go about doing this? better alternatives?

**Comment by oxalorg** [comment]
Take a look at how Hacker News does it. Simply route to the required rant using a GET parameter.

```
https://news.ycombinator.com/item?id=14578807
```

I think this would be an easy solution, or you can use those browser hashes used by SPAs (and then retrieve the required rant from your API endpoint)

**Comment by prodicus** [comment]
closed by #16
